backend/services/user.js

```diff
@@ -92,3 +93,36 @@ export async function registerUser(userData, imageFiles) {
     throw ServiceError.INVALID_USER_RECEIVED.addContext(e.stack);
   }
 }
+
+export async function updateUser(userId, user) {
```

Comment:
Same here - There's a service in main that does updateUser, can we bring the verificationStatus check to that service instead of making a new one?

Reply:
I think it was not there when I started working on this, have merged with main